src/main.rs

```diff
@@ -101,6 +107,7 @@ async fn main() -> std::io::Result<()> {
     // Init App
     HttpServer::new(move || {
         App::new()
+            .wrap(prometheus.clone())
```

Comment:
Didn't play nice with the compression middleware, so I had to put it above it. Didn't investigate why, could if we want.